transport/grpc/stream.go

```diff
@@ -75,6 +76,14 @@ func (ss *serverStream) ReceiveMessage(_ context.Context) (*transport.StreamMess
 	return &transport.StreamMessage{Body: ioutil.NopCloser(bytes.NewReader(msg))}, nil
 }
 
+func (ss *serverStream) SendHeader(headers transport.StreamHeaders) error {
+	md := make(metadata.MD, len(headers))
+	for k, vs := range headers {
+		md.Set(k, vs...)
+	}
+	return ss.stream.SendHeader(md)
```

Comment:
nit: Since `transport.StreamHeaders` and `metadata.MD` are both aliases of `map[string][]string` I think you should just be able to do a type cast
```suggestion
	return ss.stream.SendHeader(metata.MD(headers))
```